trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -1222,7 +1222,8 @@ def evaluate_hypothesis(index, hypothesis):
 
         for i, score, reason in results:
             groundedness_scores[f"statement_{i}"] = score
-            reasons_str += f"STATEMENT {i}:\n{reason['reason']}\n"
+            reason_str = reason['reason'] if 'reason' in reason else ""
```

Comment:
maybe the else case should be "<missing_reason>" or something like that?

Reply:
"missing reason would be good here".

Also please retain both the `+=` to append the reason_str for each result and the Statement counter f`STATEMENT {i}`

Reply:
@sfc-gh-jreini @epinzur I've updated the missing reason, PTAL again